public/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="shortcut icon" href="./images/shortcut/shopy.png" />
```

Comment:
favicon should have '.ico' format

suggest keep using %PUBLIC_URL%/

Reply:
done